redaxo/src/core/fragments/core/footer.php

```diff
@@ -11,3 +11,18 @@
             </ul>
         </nav>
     </footer>
+    <?php if(rex::getProperty('session_keep_alive', 0)): ?>
+    <!-- keep session alive -->
+    <script type="text/javascript">
```

Comment:
Ich glaube, ich würde das eher in der standard.js platzieren, oder was meinst du dazu @staabm?

Reply:
gerne in die extene javascript datei packen. man könnte den wert für den interval dann z.b. via `data-sess-keep-alive` o.ä. am `<body>` einstellbar machen

Reply:
Oder mit `rex_view::setJsProperty()` setzen

Reply:
https://github.com/redaxo/redaxo/blob/42ce383f550cec66e522e2238b0844c40862a12a/redaxo/src/core/backend.php#L111-L112